### ISSUE TITLE
Markup error in twisted.protocols.amp docstring

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5673]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5673|
|Type|defect|
|Created|2012-05-09 13:35:27Z|

r34350 introduced this new pyflakes error:

```
these 1 objects' docstrings are not proper epytext:
    twisted.protocols.amp.Descriptor.fromStringProto
```

due to a simple typo.

Attachments:

* [amp-epytext.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/c11/c117b64ac3f7aeab3389d9b891659078db9b4a30/bfd03897f673f18a22280b0993b2fa7782815b03.patch) (605 bytes) - added by exarkun on 2012-05-09 13:36:02Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5673 5673
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1336570527000000 1336570527000000
changetime__1336589988000000 1336589988000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_2"></a>|@itamarst set owner to @exarkun|
|-|-|

Looks good, merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [34362]) Correct a typo in the epytext markup for twisted.protocols.amp.Descriptor.fromStringProto

Author: exarkun
Reviewer: itamar
Fixes: [#5673](https://github.com/twisted/twisted/issues/5673)
